src/antlr/Parser.g

```diff
@@ -982,9 +982,30 @@ columnConstraints returns [ColumnConstraints.Raw constraints]
     ;
 
 columnConstraint returns [ColumnConstraint columnConstraint]
-    : funcName=ident '(' k=ident ')' op=relationType t=value { $columnConstraint = new FunctionColumnConstraint.Raw(funcName, k, op, t.getText()).prepare(); }
-    | funcName=ident '(' k=ident ')' { $columnConstraint = new UnaryFunctionColumnConstraint.Raw(funcName, k).prepare(); }
-    | k=ident op=relationType t=value { $columnConstraint = new ScalarColumnConstraint.Raw(k, op, t.getText()).prepare(); }
+    @init { List<String> arguments = new ArrayList<>(); }
```

Comment:
this will enable:

`NOT NULL` special treatment for not null
`JSON` unary
`SOMECONSTRAINT('x', ...)` (function)
`SOMECONSTRAINT('x') > 5` (function with relation and term)
`val int check val > 5` (scalar)